vhdbuilder/packer/windows-image.env

```diff
@@ -4,6 +4,6 @@
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-with-Containers-smalldisk:latest
 WINDOWS_2019_BASE_IMAGE_SKU=2019-Datacenter-Core-smalldisk
-WINDOWS_2019_BASE_IMAGE_VERSION=17763.1757.2102060435
+WINDOWS_2019_BASE_IMAGE_VERSION=17763.1817.2103030313
 WINDOWS_2004_BASE_IMAGE_SKU=datacenter-core-2004-with-containers-smalldisk
```

Comment:
Remove 2004?

Reply:
I can start a sperate PR to remove all code used for windows 2004 later.